src/_blog/webtransport.md

```diff
@@ -0,0 +1,252 @@
+---
+tags:
+- browser, transport
+title: WebTransport in libp2p
+description:
+date: 2022-12-15
+permalink: "/2022-12-15-libp2p-webtransport/"
+translationKey: ''
+header_image: /
+author: Marten Seemann
+---
+
+# WebTransport in libp2p
+
+<!--
+WebRTC (Browser to Server): Part 2 of Universal Browser Connectivity
+WebRTC (Browser to Browser): Part 3 of Universal Browser Connectivity
+-->
+This is the first entry in a series of posts on how libp2p achieves browser connectivity.
+
+**Table of Contents**
+
+[[toc]]
+
+## Overview
+
+Universal and seamless browser connectivity is a key goal of the [libp2p project](https://github.com/libp2p/specs). And over the course of many years, libp2p has made many strides to realize that vision. Today, we are proud to announce a major milestone that puts us much closer towards that aim:
+
+**libp2p now supports the new, bleeding-edge [WebTransport protocol](#What-is-WebTransport)!**
+
+In this article, we:
+
+* [Introduce WebTranport](#What-is-WebTransport)
+* Show what it means for apps and [how you can use it today](#Can-I-use-this-right-now) 
+* Explain [its advantages over existing solutions](#The-old-solution-WebSocket-and-its-challenges)
+* Give you a [deep dive into how it works](#Meet-WebTransport)
+* Describe the [current state of WebTransport](#What’s-the-current-state-of-WebTransport-and-where-is-it-supported) (specs and implementations)
+
+In this post, we’ll explain how we’re already using it to improve browser connectivity in libp2p.
+
+## What is WebTransport?
+
+At a high level, [WebTransport](https://www.w3.org/TR/webtransport/) is a new [transport protocol](https://osi-model.com/transport-layer/) and [Web API](https://developer.mozilla.org/en-US/docs/Web/API) currently under development by both the [Internet Engineering Task Force (IETF)](https://www.ietf.org/) and the [World Wide Web Consortium (W3C)](https://www.w3.org/).
+
+This protocol was developed [to meet these goals](https://github.com/w3c/webtransport/blob/main/explainer.md#goals):
+
+- Enable low latency communication between browsers and servers (efficiently transfer data and decrease travel time from browser to server.)
+- Have an API that supports different protocols and use cases e.g. reliable/unreliable and ordered/unordered data transmission, client-server and peer-to-peer architectures, transmitting audio/video media as well as generic data.
+- Have the same security properties as current solutions (WebSocket over TLS.)
```

Comment:
```suggestion
- Have the same security properties as current solutions (e.g., WebSocket over TLS.)
```

Lets be consistent.  If we're putting examples in paranthesis, then add "e.g.," to all of them.